cypress/support/POM/dashboard-page.ts

```diff
@@ -1,24 +1,27 @@
-class DashboardPage{
-    static elements = {
-        mainMenuItems: () => cy.get('.oxd-sidepanel-body'),
-        userIcon: () =>   cy.get('.oxd-userdropdown-tab > .oxd-icon'),
-        logout: () =>   cy.get(':nth-child(4) > .oxd-userdropdown-link')
-    }
+class DashboardPage {
+  static elements = {
+    mainMenuItems: () => cy.get(".oxd-sidepanel-body"),
+    userIcon: () => cy.get(".oxd-userdropdown-tab > .oxd-icon"),
+    logout: () => cy.get(":nth-child(4) > .oxd-userdropdown-link"),
+  };
 
-    static openPimPage(){
-        this.elements.mainMenuItems().contains('PIM').click();
-    }
+  static openPimPage() {
+    this.elements.mainMenuItems().contains("PIM").click();
+  }
 
-    static openRecruitmentPage(){
-        this.elements.mainMenuItems().contains('Recruitment').click();
-    }
+  static openRecruitmentPage() {
+    this.elements.mainMenuItems().contains("Recruitment").click();
```

Comment:
it's better to have such names as a general constant, So in this way if we have a change in one of them, no need to change them in all pages,